frontend/pages/_app.tsx

```diff
@@ -25,9 +25,15 @@ import { StakingContractDetailsProvider } from '@/context/StakingContractDetails
 import { StakingProgramProvider } from '@/context/StakingProgramProvider';
 import { StoreProvider } from '@/context/StoreProvider';
 import { SupportModalProvider } from '@/context/SupportModalProvider';
-import { SystemNotificationTriggers } from '@/context/SystemNotificationTriggers';
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      // Run polling even when window is hidden/minimized
+      refetchIntervalInBackground: true,
```

Comment:
I guess this is needed for showing notifications? I'd rather put this in particular queries, not to waste resources... or change the interval when the app is minimized from e.g. 5 sec to 30-60...